app/paths.py

```diff
@@ -1,9 +1,13 @@
 from pathlib import Path
-import os
 
-IS_IN_DOCKER = os.geteuid() == 0
+import sys
+import os
 
-STORAGE_PATH = Path('/opt/storage/') if IS_IN_DOCKER else Path(f'/home/{os.getlogin()}/.gerev/storage/')
+IS_IN_DOCKER = False if (sys.platform == "win32" or not hasattr(os, "geteuid")) else os.geteuid() == 0
```

Comment:
If think you shouldn't use if in case of boolean sentence. However there is a library that handle that:
https://pypi.org/project/jaraco.docker/

How to use:
```python
from jaraco.docker import is_docker

if is_docker():
    print("yes is docker!
```